components/hero-slider/HeroSlide.tsx

```diff
@@ -1,50 +1,50 @@
-import Link from "next/link";
+import DOMPurify from "isomorphic-dompurify";
+
 import { FC } from "react";
 
 import { HeroSlideProps } from "./slider-types";
 
 export const HeroSlide: FC<HeroSlideProps> = ({ slide }) => {
+  const sanitizedData = () => ({
+    __html: DOMPurify.sanitize(slide.title.text),
+  });
+
   return (
-    <Link href={!slide.title.text ? slide.link1 : "/"}>
-      <div className="flex items-center h-full w-full">
-        <div className="w-full md:max-w-[540px] pb-8">
-          <h2
-            className={`font-kanit text-center sm:text-left mb-6 leading-tight ${slide.title.styles}`}
+    <div className="flex items-center h-full w-full">
+      <div className="w-full pb-8">
+        <div
+          className={`text-center sm:text-left mb-6 ${slide.title.styles}`}
+          dangerouslySetInnerHTML={sanitizedData()}
```

Comment:
What's the reason for this? We should avoid using `dangerouslySetInnerHTML` as the consequences of XSS attacks on our site are high